### PR TITLE
GRW-2091 - feat(ImageBlock): added support for design system based aspect ratio

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Added storyblok datasource for aspect ratios
* Added support for design system based aspect ratio

## Justify why they are needed

Requested by design

## Jira issue(s): [GRW-2091](https://hedvig.atlassian.net/browse/GRW-2091)

Screenshot 2023-01-24 at 16 13 59

Screenshot 2023-01-25 at 10 25 40

[GRW-2091]: https://hedvig.atlassian.net/browse/GRW-2091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1442** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1442" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1442?utm_source=stack-comment).

File: apps/store/src/blocks/ImageBlock.tsx [hunk 1]
```diff
@@ -2,28 +2,37 @@ import isPropValid from '@emotion/is-prop-valid'
 import styled from '@emotion/styled'
 import { storyblokEditable } from '@storyblok/react'
 import { default as NextImage } from 'next/image'
-import { mq } from 'ui'
+import { mq, theme } from 'ui'
 import { HeadingBlock, HeadingBlockProps } from '@/blocks/HeadingBlock'
 import { ExpectedBlockType, SbBaseBlockProps, StoryblokAsset } from '@/services/storyblok/storyblok'
 import { filterByBlockType } from '@/services/storyblok/Storyblok.helpers'
 
+export type ImageAspectRatio = '1 / 1' | '2 / 1' | '3 / 2' | '4 / 3' | '5 / 4' | '18 / 9'
```
Comment (gustaveen) [inline apps/store/src/blocks/ImageBlock.tsx:10]: Should this be `16 / 9`?
```suggestion
export type ImageAspectRatio = '1 / 1' | '2 / 1' | '3 / 2' | '4 / 3' | '5 / 4' | '16 / 9'
```

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/blocks/ImageBlock.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/blocks/ImageBlock.tsx:10]: You're right 😅
Nice catch

**Pull request merged**